S1_NRB/metadata/extract.py

```diff
@@ -490,6 +545,9 @@ def meta_dict(config, target, src_ids, snap_datasets, proc_time, start, stop, co
     tups = [(ITEM_MAP[key]['suffix'], ITEM_MAP[key]['z_error']) for key in ITEM_MAP.keys()]
     z_err_dict = dict(tups)
     
+    geocorr_acc = calc_geolocation_accuracy(swath_identifier=meta['common']['swathIdentifier'], ei_tif=ei_tif,
+                                            dem_type=dem_type, etad=config['etad'])
+    
```

Comment:
`meta['common']['swathIdentifier']` is queried but does not exist before line 575.